app/components/works/add_files_component.html.erb

```diff
@@ -64,7 +68,7 @@
 
         <template data-dropzone-target='template'>
           <%= form.fields_for :attached_files, AttachedFile.new, child_index: 'TEMPLATE_RECORD' do |file_form| %>
-            <%= render Works::FileRowComponent.new(form: file_form, zip_template: true) %>
+            <%= render Works::Edit::FileRowComponent.new(form: file_form, zip_template: true) %>
           <% end %>
```

Comment:
this component just got moved to the `Edit` namespace (and was modified slightly as well)